testing/test_motors.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import context
+import unittest
+import cv2
+import roboplot.core.stepper_motors as stepper_motors
+import roboplot.core.hardware as hardware
+from roboplot.core.gpio.gpio_wrapper import GPIO
+import time
```

Comment:
Also, several of the `import`s here are unused.
Pycharm warns me about this so it's not so much of a problem, but if I was using a simpler text editor (like Max) or looking at it on GitHub, I imagine I could get quite confused looking for say the illusive open cv references!